test/js/loan_store_test.js

```diff
@@ -1,35 +1,303 @@
 var chai = require('chai');
 var expect = chai.expect;
 var loanStore;
+var scenarioStore;
 var $;
 var sinon = require('sinon');
 var mortgageCalculations = require('../../src/static/js/modules/loan-comparison/mortgage-calculations.js');
 
+var api;
+
 
 describe('Loan store tests', function() {
     before(function () {
         $ = require('jquery');
         loanStore = require('../../src/static/js/modules/loan-comparison/stores/loan-store.js');
+        scenarioStore = require('../../src/static/js/modules/loan-comparison/stores/scenario-store.js');
+        api = require('../../src/static/js/modules/loan-comparison/api.js');
     });
     
+    describe('init', function() {
+        it('should reset all loans when init', function() {
```

Comment:
I am actually just checking if reset all loans function is being called.  But sure that seems more clear.